pkg/validate/streaming.go

```diff
@@ -74,10 +74,38 @@ var _ = framework.KubeDescribe("Streaming", func() {
 			By("start container")
 			startContainer(rc, containerID)
 
-			req := createDefaultExec(rc, containerID)
+			execReq := &runtimeapi.ExecRequest{
+				ContainerId: containerID,
+				Cmd:         []string{"echo", "hello"},
+				Stdout:      true,
+				Stderr:      true,
+			}
+			req := createExec(rc, execReq)
 
 			By("check the output of exec")
-			checkExec(rc, req)
+			checkExec(rc, req, "hello\n", false)
+		})
+
+		It("runtime should support exec with -it flags", func() {
```

Comment:
You should make a different PR for exec tests. Also we should add a test for exec without -it in that PR.

Reply:
Removed the exec test, will open another PR with those tests.